mmv1/third_party/terraform/utils/provider_test.go.erb

```diff
@@ -106,8 +106,8 @@ func TestAccProviderUserProjectOverride(t *testing.T) {
 	pid := "tf-test-" + RandString(t, 10)
 	topicName := "tf-test-topic-" + RandString(t, 10)
 
-	config := BootstrapConfig(t)
-	accessToken, err := setupProjectsAndGetAccessToken(org, billing, pid, "pubsub", config)
+	config := acctest.BootstrapConfig(t)
+	accessToken, err := acctest.SetupProjectsAndGetAccessToken(org, billing, pid, "pubsub", config)
```

Comment:
Same question here

Reply:
Thanks for catching it. These changes should not in this PR.